modules/oadp-features-plugins-known-issues.adoc

```diff
@@ -49,3 +49,22 @@ You can force the reconciliation by making any minor change to the DPA itself.
 ----
 $ oc -n openshift-adp get secret/oadp-<bsl_name>-<bsl_provider>-registry-secret -o json | jq -r '.data'
 ----
+
+
+[id="openshift-adp-controller-manager-seg-fault_{context}"]
+== `openshift-adp-controller` segmentation fault
+
+The `openshift-adp-controller-manager` pod fails with a crash loop segmentation fault, when the pod crashes and is restarted indefinitely until returning a segmentation fault, when configuring a DPA with `cloudstorage` and `restic` enabled.
+
+You can have either `velero` or `cloudstorage` defined, as they are mutually exclusive fields.
+
+* If you have both `velero` and  `cloudstorage` defined, the `openshift-adp-controller-manager` will fail.
+* If you have neither `velero` nor `cloudstorage` defined, the `openshift-adp-controller-manager` will fail.
```

Comment:
IBM Style says to [avoid future tense](https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#tense).

```suggestion
You can have either `velero` or `cloudstorage` defined, as they are mutually exclusive fields.

* If you have both `velero` and  `cloudstorage` defined, the `openshift-adp-controller-manager` fails.
* If you have neither `velero` nor `cloudstorage` defined, the `openshift-adp-controller-manager` fails.
```